common/org.testeditor.dsl.common.ui/src/org/testeditor/dsl/common/ui/utils/ProjectUtils.xtend

```diff
@@ -69,4 +71,14 @@ class ProjectUtils {
 		return folder
 	}
 
+	/** get a preference node (e.g. for node "org.testeditor.tcl.dsl.Tcl") */
+	def IEclipsePreferences getPrefsNode(String node) {
+		return InstanceScope.INSTANCE.getNode(node);
```

Comment:
tried 
		val context = contextHelper.eclipseContext
		val instanceScope = context.get(InstanceScope)
but it did not work.
Got the notion for this static access from http://www.vogella.com/tutorials/EclipsePreferences/article.html#eclipse-preference-api